src/service/mercury/index.ts

```diff
@@ -724,4 +742,34 @@ export class MercuryClient {
       },
     };
   };
+
+  getAccountSubForPubKey = async (
+    publicKey: string,
+    network: NetworkNames
+  ): Promise<{ publickey: string }[]> => {
+    try {
+      const getData = async () => {
+        const urqlClient = this.mercurySession.backendClientMaker(
+          network,
+          this.mercurySession.token
+        );
+        const response = await urqlClient.query(
+          query.getAccountSubForPubKey(publicKey),
+          {}
+        );
+
+        const errorMessage = getGraphQlError(response.error);
+        if (errorMessage) {
+          throw new Error(errorMessage);
```

Comment:
in this scenario, do we want to fallback to `getAccountHistoryHorizon`? I'm not 100% sure if this `throw` will propagate up to `getAccountHistory`